modin/pandas/series.py

```diff
@@ -181,7 +182,7 @@ def __and__(self, other):
         return super(Series, new_self).__and__(new_other)
 
     def __array__(self, dtype=None):
-        return super(Series, self).__array__(dtype).flatten()
+        return super().__array__(dtype).flatten()
```

Comment:
Let's move these to a different PR.

Reply:
yeah, I just forgot to revert those when I prepared my PR, will fix.